ui/raidboss/emulator/data/CombatantJobSearch.ts

```diff
@@ -0,0 +1,133 @@
+export default class CombatantJobSearch {
+  static getJob(abilityID: number): string | undefined {
+    for (const Job in CombatantJobSearch.abilities) {
+      if (CombatantJobSearch.abilities[Job]?.includes(abilityID))
```

Comment:
As a future (not in this PR) performance / simplicity suggestion you could turn the map below into an ability -> job map and then just return a single lookup rather than a loop and an implicit loop in includes.  Could also skip the parseInt too and store the hex string.  Maybe that's just premature optimization.  :)

Reply:
Both of these are CPU vs Memory tradeoffs. I decided to opt for memory-friendly more than CPU-friendly because this isn't very often-executed code (only executed against an ability line if the combatant is a player and their job hasn't been detected yet).

The parseInt will happen upstream in LineEvent once those refactors are done.